src/ffmpeg/AVUtil/v56/libavutil_h.jl

```diff
@@ -1929,7 +1929,7 @@ const AV_TIMECODE_FLAG_24HOURSMAX = 2 |> UInt32
 const AV_TIMECODE_FLAG_ALLOWNEGATIVE = 4 |> UInt32
 # end enum AVTimecodeFlag
 
-const AVTimecode = 
+const AVTimecode =
```

Comment:
Anyone know what this is about?? Surprised it isn't erroring!

Reply:
It's parsing down to the `struct AVTreeNode`. Not sure if that's a bug or intended (strange) use of whitespace...

Reply:
It's not immediately clear what it should be, so will leave it 🤷‍♂️

Reply:
It's from here: https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/timecode.h#L41-L46

That's declaring an anonymous struct, and assigning it to `AVTimecode`.  I'm guessing that Clang.jl didn't wrap this correctly, so if this needs to be initialized in Julia, it would probably be good to change this definition to be an actual struct, matching the FFmpeg code above.

Reply:
(FWIW, I had to look up the difference between `struct` and `typedef Foo struct`, and found this explanation useful: https://stackoverflow.com/questions/612328/difference-between-struct-and-typedef-struct-in-c)

Reply:
I've left this as-is. It can be revisited when the Clang wrapper is updated